### PR TITLE
Enabled CDC-ACM on BMC_USB_OTG port

### PR DESCRIPTION
**Pull request by svenrademakers**
Violating someone else's USB VID/PID...

### CODE REVIEW

File: tp2bmc/board/tp2bmc/overlay/etc/init.d/S11bmc-otg-acm [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+setup_configuration() {
+    mount -t configfs none /sys/kernel/config > /dev/null 2>&1
+    mkdir -p $usb_gadget
+    cd $usb_gadget
+
+    echo "0x08DE" > idVendor
+    echo "0x1234" > idProduct
+    mkdir -p strings/0x409
```
Comment (ruslashev) [inline tp2bmc/board/tp2bmc/overlay/etc/init.d/S11bmc-otg-acm:28]: What's the reason behind number 0x409?

Comment (svenrademakers) [reply tp2bmc/board/tp2bmc/overlay/etc/init.d/S11bmc-otg-acm:28]: With 0x409 you declare that the strings are in English

File: tp2bmc/board/tp2bmc/linux_defconfig [hunk 2]
```diff
@@ -168,12 +169,14 @@ CONFIG_USB_UAS=y
...
 CONFIG_USB_SUNXI_UDC0=y
 CONFIG_USB_CONFIGFS=y
+CONFIG_USB_CONFIGFS_ACM=y
 CONFIG_USB_CONFIGFS_UEVENT=y
 CONFIG_USB_CONFIGFS_SERIAL=y
 CONFIG_USB_CONFIGFS_RNDIS=y
 CONFIG_USB_CONFIGFS_MASS_STORAGE=y
 CONFIG_USB_CONFIGFS_F_FS=y
 CONFIG_USB_CONFIGFS_F_HID=y
+CONFIG_USB_CONFIGFS_F_ACM=y
```
[the remark this answers is not in the document]
Comment (svenrademakers) [reply tp2bmc/board/tp2bmc/linux_defconfig:179]: actually you are right, it should have been `CONFIG_USB_F_ACM,` but this option is acually selected when you configure `USB_CONFIGFS_ACM`: https://github.com/torvalds/linux/blob/3f7168591ebf7bbdb91797d02b1afaf00a4289b1/drivers/usb/gadget/Kconfig#L251

[the remark this answers is not in the document]
Comment (svenrademakers) [reply tp2bmc/board/tp2bmc/linux_defconfig:179]: Hence i will remove this declaration

**Review by ruslashev — APPROVED** [review]
Using the '???' vendor ID seems weird, but other than that LGTM

**Pull request merged**